controllers/messages_controller.py

```diff
@@ -0,0 +1,39 @@
+import json
+import jwt
+import logging
+
+
+import requests
+from structlog import wrap_logger
+
+from config import Config
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def create_message(subject, body):
+    logger.debug('Creating secure message')
+
+    message = {
+        'msg_from': "BRES",
+        'msg_to': ["f62dfda8-73b0-4e0e-97cf-1b06327a6712"],
+        'subject': subject,
+        'body': body,
+        'thread_id': "",
+        'collection_case': "",
+        'survey': "cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87",
+        'ru_id': "c614e64e-d981-4eba-b016-d9822f09a4fb"}
+
+    url = Config.SECURE_MESSAGE_SERVICE + '/message/send'
```

Comment:
Can we target version 2, instead of one as that only handles BRES and all the new functionality needs to work with version 2?

Reply:
As talked about, this needs to be a future piece of work as the current inbox also only uses the v1 code.